src/core/struct_tree.js

```diff
@@ -13,29 +13,49 @@
  * limitations under the License.
  */
 
-import { Dict, isName, Name, Ref } from "./primitives.js";
+import { Dict, isName, Name, Ref, RefSetCache } from "./primitives.js";
 import { stringToPDFString, warn } from "../shared/util.js";
 import { NumberTree } from "./name_number_tree.js";
 
 const MAX_DEPTH = 40;
 
 const StructElementType = {
-  PAGE_CONTENT: "PAGE_CONTENT",
-  STREAM_CONTENT: "STREAM_CONTENT",
-  OBJECT: "OBJECT",
-  ELEMENT: "ELEMENT",
+  PAGE_CONTENT: 0,
```

Comment:
May I suggest that we start the numbering in an "enum" like this from `1` instead?